db/seeds/production.rb

```diff
@@ -0,0 +1,421 @@
+)
+
+Review.create!(title: "My new spot",
+               rating: 5,
+               body: "Came here to watch a football game... which was a bad idea
+                because there are only 2 tiny TV's. But I would love to come here
+                to hang out with my friends. I want this to be my spot. So cozy,
+                warm, and reasonably priced.\n
+                My friends and I got the chicken BLT and a burger, plus sweet
+                potato fries. Ate every bite, totally tasty.\n
+                Its just a little gem in a big city.",
+               user_id: 4,
+               bar_id: 4
+)
+
+Review.create!(title: "My new spot",
+               rating: 3,
+               body: "This place was alright. I got the loaded nachos. It was
+               actually really good. I added the gaucamole and it was worth it.
+               Nachos are a great plate to share and I did share it with 3 other
+               people. I could not even finish it, it was very much loaded.\n
+               Different variety of beers. i got the watermelon kind, but it was
+               not great. Just stick to something you know. Server did not know
+               much about what was on draft and service was not that great, but
+               food was good.",
+               user_id: 5,
+               bar_id: 4
+)
+
+Review.create!(title: "Good food, spotty service",
+               rating: 4,
+               body: "Food very good.  We had a party of ten and all the food
+               came out together - and still warm. Service was good not great.
+               The menu had a wide variety so had something even for picky
+               eaters. The have a nice outside eating area but doesn't
+               accommodate a large party.",
+               user_id: 5,
+               bar_id: 5
+)
+
+Review.create!(title: "Get better help!",
+               rating: 1,
+               body: "I usually like this place but the past few times have been
+                horrible. The servers are inexperienced and are not attentive at
+                all. I experienced the worst service last week when I was served
+                a flat beer (after waiting 20 min). Because the server didn't
+                ever come back to the table, I went to the bar and asked the
+                bartender (the balding man with glasses) if I could get something
+                else since it was flat. He said the keg had just turned over so I
+                had to deal with it and would not give me another beer
+                (and he walked away) The worst customer service in the 2 years I
+                have been in Boston. I used to be a bartender - customers should
+                always be taken care of no matter what.",
+               user_id: 6,
+               bar_id: 5
+)
+
+Review.create!(title: "Delicioso!",
+                rating: 1,
+                body: "We 'stumbled' upon this place seeing that getting into toro
```

Comment:
Line is too long. [82/80]<br>Align the elements of a hash literal if they span more than one line.